src/views/Pools/components/CakeVaultCard/VaultCardActions/index.tsx

```diff
@@ -55,7 +55,7 @@ export const IfoVaultCardAvgBalance = () => {
       <Flex flexDirection="column" pb="16px">
         <Balance fontSize="20px" bold value={cakeAsNumberBalance} decimals={5} />
         <Text fontSize="12px" color="textSubtle" display="flex">
-          {creditsDollarValue ? (
+          {creditsDollarValue !== undefined ? (
             <Balance
```

Comment:
Imho Number.isFinite(creditsDollarValue) is more descriptive

Reply:
<img width="552" alt="Screen Shot 2022-01-12 at 14 59 01" src="https://user-images.githubusercontent.com/97418926/149086954-ad5621bd-3791-489c-89e9-46e18cf23617.png">

Thanks. Using type check in this case would be enough.

Reply:
True, but if logic of that method changes in the future we would be on the safe side if we handle more cases

Reply:
At this moment, we can use type check as a contract. In the future, if the method is changed, the contract would be broken. Type Check could notify that. I think it would be safe.